src/components/withPrepareCentralPaneScreen/index.native.tsx

```diff
@@ -0,0 +1,27 @@
+import type {ComponentType, ForwardedRef, RefAttributes} from 'react';
+import React from 'react';
+import getComponentDisplayName from '@libs/getComponentDisplayName';
+import FreezeWrapper from '@libs/Navigation/FreezeWrapper';
+
+/**
+ * This HOC is dependent on platform. On native platforms screens that aren't already dipslayed in the navigation stack should be freezed to prevent from unnecessary rendering.
```

Comment:
```suggestion
 * This HOC is dependent on the platform. On native platforms, screens that aren't already displayed in the navigation stack should be frozen to prevent unnecessary rendering.
```